web/pom.xml

```diff
@@ -109,6 +116,42 @@
         <artifactId>datumbox-framework-lib</artifactId>
         <version>0.8.1</version>
     </dependency>
+	<dependency>
+		<groupId>com.github.kalletlak</groupId>
```

Comment:
before merging needs to be updated to com.github.cbioportal

Reply:
sure, will do it once https://github.com/cBioPortal/session-service/pull/22 is merged